### ISSUE TITLE
[BITV] Add title to Actions (NcButton) component 

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#35860]
## Problem

To show a tooltip on a button (inside of the actions) we currently use v-tooltip. An example can be found `files_sharing` app: https://github.com/nextcloud/server/blob/master/apps/files_sharing/src/views/SharingInherited.vue#L36. For now `title` attribute won't passed to  NcButton which is inside of NcActions: https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcActions/NcActions.vue#L933. The main issue is: https://github.com/nextcloud/server/issues/35654.

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to the button (inside of the actions) component that will render as `title` attribute on the HTML element.
